src/main/java/step1/domain/Operator.java

```diff
@@ -0,0 +1,36 @@
+package step1.domain;
+
+import step1.config.ErrorMessage;
+
+import java.util.function.BiFunction;
+
+public enum Operator implements OperatorType {
+    ADD("+", (number1, number2) -> number1 + number2),
+    SUBTRACT("-",(number1, number2) -> number1 - number2),
+    MULTIPLY("*",(number1, number2) -> number1 * number2),
+    DIVIDE("/",(number1, number2) -> number1 / number2);
+
+    private final String symbol;
+    private final BiFunction<Integer, Integer, Integer> expression;
+
+    Operator(String symbol, BiFunction<Integer, Integer, Integer> expression) {
+        this.symbol = symbol;
+        this.expression = expression;
+    }
+
+    public static Operator find(String text) {
+
+        for (Operator operation : Operator.values()) {
+            if (operation.symbol.equals(text)) {
```

Comment:
```suggestion
            if (operation.matches(text)) {
```
이전 동등한지 비교하는 static 메소드가 사라졌네요 👍
지금 구조에서도 오퍼레이션에게 메시지를 보내도록 메소드를 만들어보면 좋을 것 같습니다.